### PR TITLE
CODEOWNERS: Update following latest changes in the code

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**

### CODE REVIEW

**Review by alban — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -13,18 +13,54 @@
...
 /examples/runc-hook/ @alban
 /gadget-container/hooks/ @alban
-/gadgets/ @mauriciovasquezbernal
+
+# gadgets
+/gadgets/audit_seccomp/ @alban
+/gadgets/ci/ @mauriciovasquezbernal
+/gadgets/deadlock/ @alban
+/gadgets/fsnotify/ @alban
+/gadgets/profile_blockio/
```
Comment (burak-ok) [inline CODEOWNERS:34]: Someone is missing here (if this wasn't intended)

**Review by burak-ok — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (mauriciovasquezbernal) [reply CODEOWNERS]: Good catch, thanks!

**Review by flyth — APPROVED** [review]

File: CODEOWNERS [hunk 2]
```diff
@@ -66,9 +100,30 @@
...
+# operators
 /pkg/operators/ @flyth
+/pkg/operators/btfgen/ @mauriciovasquezbernal
+/pkg/operators/cli/ @flyth
+/pkg/operators/combiner/ @blanquicet
+/pkg/operators/common/ @burak-ok
+/pkg/operators/ebpf/ @alban @mauriciovasquezbernal @flyth
+/pkg/operators/filter/ @flyth
+/pkg/operators/formatters/ @flyth @mauriciovasquezbernal
 /pkg/operators/kubeipresolver/ @burak-ok
```
Comment (mqasimsarfraz) [inline CODEOWNERS:113]: I can also review changes related to `kubeipresolver`

File: CODEOWNERS [hunk 3]
```diff
@@ -66,9 +100,30 @@
 /pkg/kallsyms/ @alban
 /pkg/kfilefields/ @alban @mqasimsarfraz
 /pkg/networktracer/ @alban @blanquicet
+
+# operators
 /pkg/operators/ @flyth
-/pkg/operators/kubeipresolver/ @burak-ok
+/pkg/operators/btfgen/ @mauriciovasquezbernal
+/pkg/operators/cli/ @flyth
```
Comment (blanquicet) [inline CODEOWNERS:107]: I can help reviewing also the cli.

File: CODEOWNERS [hunk 4]
```diff
@@ -13,18 +13,54 @@
 /cmd/common/ @flyth
 /cmd/ig/ @blanquicet
 /Dockerfiles/ @eiffel-fl @mqasimsarfraz
+
+# docs
```
Comment (blanquicet) [inline CODEOWNERS:17]: And also with `docs/reference`.

**Review by blanquicet — APPROVED** [review]

**Pull request merged**